src/main/java/org/programmers/springbootbasic/voucher/repository/FileVoucherRepository.java

```diff
@@ -0,0 +1,80 @@
+package org.programmers.springbootbasic.voucher.repository;
+
+import org.programmers.springbootbasic.voucher.model.Voucher;
+import org.programmers.springbootbasic.voucher.model.VoucherType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Repository;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
+@Profile("local")
+public class FileVoucherRepository implements VoucherRepository {
+    private static final Logger logger = LoggerFactory.getLogger(FileVoucherRepository.class);
+
+    private final File file = new File("voucher_file_database.csv");
+    VoucherType voucherType;
```

Comment:
```suggestion
    private VoucherType voucherType;
```

특별한 경우가 아니면 default 접근제한자 보다는 private를 시작으로 필요시에 푸는 것이 좋을 것 같아요.